test/integration/active_record_searchable_test.rb

```diff
@@ -108,6 +110,7 @@ def setup
         ActiveRecordArticle.create! :title => 'Test'
 
         assert_raise Search::SearchRequestFailed do
+          puts "\n** INFO: This next query is expected to fail, so ignore any messages about a failed query"
```

Comment:
During this query, the console prints out a message about a failed query. This `puts` will hopefully make it more clear that this is expected.